jfr-daemon/src/main/java/com/newrelic/jfr/daemon/app/RemoteEntityBridge.java

```diff
@@ -76,11 +76,30 @@ public RemoteEntityBridge() {}
 
   Optional<String> getRemoteEntityGuid(MBeanServerConnection connection)
       throws MalformedObjectNameException {
+    final Map<String, String> metadata = getLinkingMetadata(connection);
+    return Optional.ofNullable(metadata.get(AttributeNames.ENTITY_GUID));
+  }
+
+  Optional<String> getRemoteEntityName(MBeanServerConnection connection) {
```

Comment:
I made the `getRemoteEntityGuid` method package private for testing purposes. I think that if you want to have another accessor, you should make it public and add javadoc.

Reply:
Done